com.github.bvschaik.julius.json

```diff
@@ -0,0 +1,28 @@
+{
+    "id" : "com.github.bvschaik.julius",
+    "runtime" : "org.freedesktop.Platform",
+    "runtime-version" : "23.08",
+    "sdk" : "org.freedesktop.Sdk",
+    "command" : "julius",
+    "finish-args" : [
+        "--filesystem=home",
+        "--share=ipc",
+        "--device=dri",
+        "--socket=wayland",
+        "--socket=fallback-x11",
+        "--socket=pulseaudio"
+    ],
+    "modules" : [
+        {
+            "name" : "julius",
+            "buildsystem" : "cmake-ninja",
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://github.com/bvschaik/julius.git",
+                    "branch" : "master"
+                }
+            ]
```

Comment:
Looking closer at the build output, I see this happening:
1. Git repo is checked out for the build as usual
2. flatpak-builder is invoked
3. Git repo is checked out _again_ by the builder

This means that if you do some changes locally, then build a .flatpak, your local changes are not used at all because the sources it builds are the latest master from github... how can we make it use the local sources?

Reply:
You're right, I've opened https://github.com/bvschaik/julius/pull/726 to fix this